wire-ios-protos/WireProtos.xcodeproj/project.pbxproj

```diff
@@ -394,6 +394,7 @@
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
 				INFOPLIST_FILE = ProtosTests/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
```

Comment:
For some reason this was necessary to prevent this target from having iOS 18 as a minimum deployment target. This wasn't necessary in earlier Xcode. I considered getting this working via the xconfig files but this seems to be a bit of a mess currently so went with the simpler solution.